tests/unit/sceneUpdateTests.cpp

```diff
@@ -12,147 +12,160 @@ using namespace Tangram;
-
-    REQUIRE(!sceneString.empty());
-
-    REQUIRE(SceneLoader::loadConfig(sceneString, scene->config()));
+TEST_CASE("Apply scene update to a nested map entry") {
+    // Setup.
+    Scene scene;
+    REQUIRE(SceneLoader::loadConfig(sceneString, scene.config()));
+    // Add an update.
+    std::vector<SceneUpdate> updates = {{"nest.map.a", "new_value"}};
+    // Apply scene updates, reload scene.
+    SceneLoader::applyUpdates(scene, updates);
+    const Node& root = scene.config();
+    REQUIRE(root["nest"]["map"]["a"].Scalar() == "new_value");
+}
 
-    Node& root = scene->config();
+TEST_CASE("Apply scene update to a sequence node") {
+    // Setup.
+    Scene scene;
+    REQUIRE(SceneLoader::loadConfig(sceneString, scene.config()));
+    // Add an update.
+    std::vector<SceneUpdate> updates = {{"seq", "new_value"}};
+    // Apply scene updates, reload scene.
+    SceneLoader::applyUpdates(scene, updates);
+    const Node& root = scene.config();
+    REQUIRE(root["seq"].Scalar() == "new_value");
+}
 
-    SceneLoader::parseGlobals(root["global"], scene);
-    SceneLoader::applyGlobalRefUpdates(root, scene);
-    SceneLoader::applyGlobalProperties(root, scene);
+TEST_CASE("Apply scene update to a nested sequence node") {
+    // Setup.
+    Scene scene;
+    REQUIRE(SceneLoader::loadConfig(sceneString, scene.config()));
+    // Add an update.
+    std::vector<SceneUpdate> updates = {{"nest.seq", "new_value"}};
+    // Apply scene updates, reload scene.
+    SceneLoader::applyUpdates(scene, updates);
+    const Node& root = scene.config();
+    REQUIRE(root["nest"]["seq"].Scalar() == "new_value");
+}
 
-    REQUIRE(root["cameras"]["iso-camera"]["active"].Scalar() == "true");
-    REQUIRE(root["cameras"]["perspective-camera"]["active"].Scalar() == "false");
+TEST_CASE("Apply scene update to a new map entry") {
+    // Setup.
+    Scene scene;
+    REQUIRE(SceneLoader::loadConfig(sceneString, scene.config()));
+    // Add an update.
+    std::vector<SceneUpdate> updates = {{"map.c", "new_value"}};
+    // Apply scene updates, reload scene.
+    SceneLoader::applyUpdates(scene, updates);
+    const Node& root = scene.config();
+    REQUIRE(root["map"]["c"].Scalar() == "new_value");
+}
 
-    // Update 1 - change 1 of the global values and explicitly set a value for the other
-    updates.push_back({"global.isoactive", "false"});
-    updates.push_back({"cameras.perspective-camera.active", "true"});
-    // Tangram apply scene updates, reload the scene
-    SceneLoader::applyUpdates(*scene, updates);
-    root = scene->config();
-    SceneLoader::parseGlobals(root["global"], scene);
-    SceneLoader::applyGlobalRefUpdates(root, scene);
-    SceneLoader::applyGlobalProperties(root, scene);
-    REQUIRE(root["cameras"]["iso-camera"]["active"].Scalar() == "false");
-    REQUIRE(root["cameras"]["perspective-camera"]["active"].Scalar() == "true");
+// This was previously enforced but it didn't seem clearly useful or desirable,
```

Comment:
This test fails with the current code. My point here is that I don't think this is a bad thing to allow. There's not a clear parallel with tangram-js to compare with since the update interfaces are totally different.